src/scenario.jl

```diff
@@ -89,7 +89,7 @@ function run_scenarios(
     setup()
 
     # Sort RCPs so the dataframe order match the output filepath
-    RCP = Base.sort(RCP)
+    RCP = sort(RCP)
```

Comment:
Was this intentional?

Reply:
Yes. This is how it is on the main branch currently. I had changed to `Base.sort(RCP)` in #689 but that is not necessary anymore.